listenbrainz/db/feedback.py

```diff
@@ -45,66 +47,111 @@ def delete(feedback: Feedback):
         )
 
 
-def get_feedback_by_user_id(user_id: int):
-    """ Get a list of recording feedbacks given by the user.
+def get_feedback_for_user(user_id: int, limit: int, offset: int, score: int = None):
+    """ Get a list of recording feedbacks given by the user
```

Comment:
You should add something here to explicitly say that they're returned in newest-to-oldest order.

Reply:
Added.